ogc/display-wfs-layer/README.md

```diff
@@ -16,7 +16,7 @@ Pan and zoom to see features within the current map extent.
 
 1. Create a `WfsFeatureTable` with a URL.
 2. Create a `FeatureLayer` from the feature table and add it to the map.
-3. Add a `NavigationChangedListener` to the map view and listen for a `NavigationChangedEvent`. Check if it `!isNavigating()` to detect when the user has stopped navigating the map.
+3. Add a `NavigationChangedListener` to the map view to listen for a `NavigationChangedEvent`. Check if it `isNavigating()` to detect when the user has stopped navigating the map.
```

Comment:
think it should be the following (as the original)?
```suggestion
3. Add a `NavigationChangedListener` to the map view to listen for a `NavigationChangedEvent`. Check if it `!isNavigating()` to detect when the user has stopped navigating the map.
```

Reply:
I personally think the `!` operator is just easy to miss here and can be confusing. If you check `isNavigating()` you either get true or false, and then the user can deduct whether naviation has stopped or is still going. Therefore it might not be necessary to do `!` even though we do it in the code?